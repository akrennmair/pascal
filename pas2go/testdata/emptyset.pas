program test;

var s : set of integer;

begin
	s := [ ]
end.
