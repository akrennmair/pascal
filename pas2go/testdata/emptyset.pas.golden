package main

import (
	"github.com/akrennmair/pascal/pas2go/system"
)

var _ = system.Write

// program test
func main() {
	var (
		s system.SetType[int]
	)
	_ = s

	system.SetAssign(&s, system.Set[int]())
}
